projects/plugins/jetpack/extensions/plugins/ai-assistant-plugin/components/seo-assistant/use-title-step.tsx

```diff
@@ -111,7 +124,28 @@ export const useTitleStep = ( {
 
 			// we only generate if options are empty
 			if ( newTitles.length === 0 || prevStepHasChanged ) {
-				newTitles = await getTitles();
+				try {
+					newTitles = await getTitles();
+					setHasFailed( false );
+				} catch {
+					setHasFailed( true );
+
+					addMessage( {
+						content: createInterpolateElement(
```

Comment:
I know we're still debating about the error styling, maybe we can just continue with the "chat mood" and let the user know some error happened and he can try again and blabla, but pointing to the regenerate button on the assistant? This is @grbicsanja 's expertise/call, I just thought I'd mention it.